index.bs

```diff
@@ -1610,7 +1886,243 @@ level APIs. The list above includes interface classes for which such high level
+            1. If |interfaceIndex| is equal to <code>-1</code>, return <code>false</code>.
+            1. Return |device|.{{USBDevice/[[claimedInterface]]}}[|interfaceIndex|].
+</dl>
+
+
+
+## The USBAlternateInterface Interface ## {#usbalternateinterface-interface}
+
+<xmp class="idl">
+  [
+    Exposed=(DedicatedWorker,SharedWorker,Window),
+    SecureContext
+  ]
+  interface USBAlternateInterface {
+    constructor(USBInterface deviceInterface, octet alternateSetting);
+    readonly attribute octet alternateSetting;
+    readonly attribute octet interfaceClass;
+    readonly attribute octet interfaceSubclass;
+    readonly attribute octet interfaceProtocol;
+    readonly attribute DOMString? interfaceName;
+    readonly attribute FrozenArray<USBEndpoint> endpoints;
+  };
+</xmp>
+
+Instances of {{USBAlternateInterface}} are created with the <a>internal
+slots</a> described in the following table:
+
+<table class="data" dfn-for="USBAlternateInterface" dfn-type="attribute">
+  <tr>
+    <th><a>Internal Slot</a></th>
+    <th>Initial Value</th>
+    <th>Description (non-normative)</th>
+  </tr>
+  <tr>
+    <td><dfn>\[[interface]]</dfn></td>
+    <td>&lt;always set in prose></td>
+    <td>
+      The {{USBInterface}} object <a>this</a> belongs to.
+    </td>
+  </tr>
+  <tr>
+    <td><dfn>\[[endpoints]]</dfn></td>
+    <td>An empty <a>sequence</a> of {{USBEndpoint}}</td>
+    <td>
+      All endpoints exposed by this interface.
+    </td>
+  </tr>
+  <tr>
+    <td><dfn>\[[alternateSetting]]</dfn></td>
+    <td>&lt;always set in prose></td>
+    <td>
+      The alternate setting for this interface.
+    </td>
+  </tr>
+</table>
+
+<div algorithm="find a list of endpoint descriptors">
+    To <dfn>find a list of endpoint descriptors</dfn> with the given |alternateSetting|
+    , |interfaceNumber|, and |configurationValue|, performing the following steps:
+    1. Let |descriptors| as a result of <a>finding a list of descriptors for a configuration</a>
+        with |configurationValue|.
+    1. Let |endpointDescriptors| as an empty <a>list</a>.
+    1. Let |index| be 0.
+    1. While |index| is smaller than the size of |descriptors|:
+        1. Let |descriptor| be |descriptors|[|index|]
+        1. If <code>bDescriptorType</code> of |descriptor| is not equal to <code>INTERFACE</code>,
+            advance |index| by 1 and continue.
+        1. if <code>bInterfaceNumber</code> of |descriptor| is not equal |interfaceNumber|,
+            advance |index| by 1 and continue.
+        1. if <code>bAlternateSetting</code> of |descriptor| is not equal |alternateSetting|,
+            advance |index| by 1 and continue.
+        1. If <code>bNumEndpoints</code> is equal to 0, return |endpointDescriptors|.
+        1. Let |numEndpoints| be <code>bNumEndpoints</code> of |descriptor|.
+        1. Let |indexEndpoints| be 0.
+        1. Let |offset| be |index| + 1
+        1. While |indexEndpoints| is smaller than |numEndpoints|:
```

Comment:
Use "less" instead of "smaller".

Reply:
Updated in https://github.com/WICG/webusb/pull/212/commits/e1b37d2a2893a75f3823dc2e2fe7635c22c1bfc4.